src/java.base/share/classes/java/lang/invoke/MethodHandleProxies.java

```diff
@@ -345,7 +345,7 @@ private static byte[] createTemplate(ClassLoader loader, ClassDesc proxyDesc, Cl
                         ClassLoaders.platformClassLoader() : loader)))
                         .build(proxyDesc, clb -> {
             clb.withSuperclass(CD_Object)
-               .withFlags((PreviewFeatures.isEnabled() ? ACC_IDENTITY  : 0) | ACC_FINAL | ACC_SYNTHETIC)
+               .withFlags(ACC_SUPER | ACC_FINAL | ACC_SYNTHETIC)
```

Comment:
I presume this change is because the classfile API will be able to filter out ACC_SUPER based on the target version?

Reply:
No, because the old code is wrong - the flags are no-ops if the class file version is not latest release's preview (by default it is latest release.0)